packages/@react-stately/table/src/TableCollection.ts

```diff
@@ -19,7 +21,12 @@ interface GridCollectionOptions {
 
 const ROW_HEADER_COLUMN_KEY = 'row-header-column-' + Math.random().toString(36).slice(2);
 
-function buildHeaderRows<T>(keyMap: Map<Key, GridNode<T>>, columnNodes: GridNode<T>[]): GridNode<T>[] {
+/** @private */
+export function buildHeaderRows<T>(keyMap: Map<Key, GridNode<T>>, columnNodes: GridNode<T>[]): GridNode<T>[] {
```

Comment:
I am not sure about exporting this. I need it for the new TableCollection implementation, which currently lives in the react-aria-components package. I suppose I could copy paste it but 🤷

Reply:
if it was copy pasted to the react-aria-components, would this copy in stately ~go away~ stop being used if we replaced our table implementation with the react-aria-component?

if so, then I think I'm more ok with copying it

Reply:
We could also un-export it later if we move the new implementation into here.